ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -604,3 +604,63 @@ literals:
 
 # Added 4/12/20
 - supply_air_damper_percentage_sensor
+
+# Added 9/8/20 for SG-SIN-MBC2
+- chilled_return_water_flowrate_sensor
+- device_fault_status
```

Comment:
when would it not be a device fault? isnt that implied? whats the difference between fault_status and device_fault_status? seems ambiguous